ui-tests/src/test/java/org/openmrs/reference/ContactInfoTest.java

```diff
@@ -36,9 +35,9 @@ public void setUp() throws Exception {
     }
 
     @Test
-    @Ignore
     @Category(BuildTests.class)
     public void contactInfoTest() throws Exception {
+    	
```

Comment:
Not sure why you added the empty line `LN-40` which was not there before. To have consistency within our code base, I would recommend you omit the line. Cross check other existing test methods within the other test and find out whether they do have this empty.

Reply:
For code quality and readability we might need this line since we have been ephasizing this clean code convention style